product_harmonized_system/models/hs_code.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2011-2021 Akretion France (http://www.akretion.com)
+# Copyright 2009-2021 Noviat (http://www.noviat.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# @author Luc de Meyer <info@noviat.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from textwrap import shorten
+
+from odoo import api, fields, models
+
+
+class HSCode(models.Model):
+    _name = "hs.code"
+    _description = "H.S. Code"
+    _order = "local_code"
+    _rec_name = "local_code"
+
+    hs_code = fields.Char(
+        string="H.S. Code",
+        compute="_compute_hs_code",
+        readonly=True,
+        help="Harmonized System code (6 digits). Full list is "
+        "available from the World Customs Organisation, see "
+        "http://www.wcoomd.org",
+    )
+    description = fields.Char(
+        translate=True, help="Short text description of the H.S. category"
+    )
+    local_code = fields.Char(
+        required=True,
+        help="Code used for the national Import/Export declaration. "
+        "The national code starts with the 6 digits of the H.S. and often "
+        "has a few additional digits to extend the H.S. code.",
+    )
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company",
+        string="Company",
+        # by default, company_id=False on this object
```

Comment:
Why is this column default removed and the [documentation](https://www.odoo.com/documentation/15.0/es/developer/howtos/company.html#default-company) is not followed?